### DISCUSSION TITLE
Feature Request: Boolean outputs

### DISCUSSION THREAD

**Discussion by AvivPeledTalon** [community/community#20057] [Actions]
My team started using a lot of boolean outputs and if conditions for finer control over the flow of the runs. 

For example, setting an output to 'false' if some tests failed (or 'true' if they passed), and checking it in a subsequent job (let's say retry tests job) instead of failing the test running step. 

We like this approach better than continue-on-error since we have more insight into what happened and more control over the next steps we should take.

Currently we have to check if the output value is equal to the string 'true' rather than just using boolean logic which is a little messier than it should be. 

**TLDR: It would be really nice to be able to set an output's type to boolean and have it auto cast 'true' and 'false' to the corresponding boolean values.**
